addons/web/static/src/views/fields/daterange/daterange_field.js

```diff
@@ -34,8 +34,8 @@ export class DateRangeField extends Component {
                                 ? localization.dateTimeFormat
                                 : localization.dateFormat,
                         },
-                        startDate: window.moment(this.formattedStartDate),
-                        endDate: window.moment(this.formattedEndDate),
+                        startDate: window.moment(this.formattedStartDate || window.moment()),
+                        endDate: window.moment(this.formattedEndDate || window.moment()),
```

Comment:
You can remove this commit from this PR as it will now be done at https://github.com/odoo/odoo/pull/97810